content/influxdb/v2.4/reference/release-notes/influx-cli.md

```diff
@@ -8,112 +8,142 @@ menu:
     name: influx CLI 
 ---
 
-## v2.3 [2022-04-08]
+## v2.4.0
 
-## Features
+### Features
+
+- Set membership type to member or owner with
+  [`influx org members add`](/influxdb/v2.4/reference/cli/influx/org/members/add/).
+- Add the [InfluxQL Shell (REPL)](/influxdb/v2.4/reference/cli/influx/v1/shell/).
```

Comment:
looks like broken link, maybe just cause that new file is in another PR?

Reply:
Yeah, this will be fixed with #4263.